=== command
CI="true" GITHUB_ACTIONS="true" GITHUB_EVENT_NAME="pull_request" GITHUB_REPOSITORY="project_name/project_name" GITHUB_RUN_ID="35" GITHUB_ACTOR="some_test_username" GITHUB_REF="some/branch-name" GITHUB_EVENT_PATH="<MASKED>" SEMGREP_APP_TOKEN="fake-key-from-tests" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep ci --no-force-color --no-suppress-errors
=== end of command

=== exit code
2
=== end of exit code

=== stdout - plain


┌────────────────┐
│ Debugging Info │
└────────────────┘

  SCAN ENVIRONMENT
  versions    - semgrep <MASKED> on python <MASKED>
  environment - running in environment github-actions, triggering event is
                pull_request
Fixing git state for github action pull request
Not on head ref: 7b5cda417c780f1f96c888e1d7bd062f46df236e; checking that out now.

  CONNECTION
Using b903231925961ac9d787ae53ee0bd15ec156e689 as the merge-base of 81af3f0c528f4206d48f2f1d1a0ada5fa9e01f38 and 7b5cda417c780f1f96c888e1d7bd062f46df236e
  Initializing scan (deployment=org_name, scan_id=12345)
  Enabled products: Code, Supply Chain

┌─────────────┐
│ Scan Status │
└─────────────┘
  Scanning 2 files tracked by git with 4 Code rules:
  Scanning 2 files with 4 python rules.
  Current version has 3 findings.

Creating git worktree from 'b903231925961ac9d787ae53ee0bd15ec156e689' to scan baseline.
  Will report findings introduced by these commits (may be incomplete for shallow checkouts):
    * 7b5cda4 commit #4
    * 5b37988 commit #3



┌─────────────┐
│ Scan Status │
└─────────────┘
  Scanning 1 file tracked by git with 1 Code rule:
  Scanning 1 file.
name 'matches_by_rule' is not defined
Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/wrapper.py", line 37, in wrapper
    func(*args, **kwargs)
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/ci.py", line 436, in ci
    ) = semgrep.run_scan.run_scan(
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/run_scan.py", line 675, in run_scan
    matches_by_rule.kept,
NameError: name 'matches_by_rule' is not defined. Did you mean: 'rule_matches_by_rule'?

=== end of stdout - plain

=== stderr - plain

=== end of stderr - plain